docs/cli.rst

```diff
@@ -0,0 +1,150 @@
+
+The `dataset_dict` option is a JSON object in the following format:
+
+.. code-block:: shell
+
+    $ refinebio download-dataset --dataset-dict '{"<Experiment 1 Accession Code>": ["<Sample 1 Accession Code>", "<Sample 2 Accession Code>"], "<Experiment 2 Accession Code>": ["<Sample 3 Accession Code>", "<Sample 4 Accession Code>"]}'
+
+You can also pass in other optional command options to alter the Dataset itself and to alter how the download process works.
+
+* `path` is the path that the Dataset will be downloaded to. You specify a path to a zip file or a directory. If you pass in a path to a directory, the name of the zip file will be automatically generated in the format `dataset-<dataset_id>.zip`. By default, `path` will be set to the current directory.
+
+* `aggregation` can be used to change how the Dataset is aggregated. The default is "EXPERIMENT", and the other available choices are "SPECIES" and "ALL". For more information about Dataset aggregation check out `Aggregations`_.
+
+* `transformation` can be used to change the transformation of the Dataset. The default is "NONE", and the other available choices are "MINMAX" and "STANDARD". For more information on Dataset transformation check out `Gene transformations`_. 
+
+* `skip-quantile-normalization` can be used to choose whether or not quantile normalization is skipped for RNA-seq Samples. For more information check out `Quantile normalization`_.
+
+* `extract` can be used to choose whether the downloaded zip file should be automatically extracted. It will automatically extract to the same location that you passed in as `path`. So if `path` is a zip file: `./path/to/dataset.zip` it will be extracted to the dir `./path/to/dataset/`, if `path` is a dir: `./path/to/dir/` it will be extracted to `./path/to/dir/[generated-file-name]/`. By default, `extract` is False. 
+
+* `prompt` can be used to choose whether or not you should be prompted before downloading if the Dataset zip file is larger than 1 gigabyte. By default, `prompt` is True.
+
+.. _Aggregations: https://refinebio-docs.readthedocs.io/en/latest/main_text.html?highlight=aggregation#aggregations 
+
+.. _Gene transformations: https://refinebio-docs.readthedocs.io/en/latest/main_text.html?highlight=quantile#gene-transformations
+
+.. _Quantile normalization: https://refinebio-docs.readthedocs.io/en/latest/main_text.html?highlight=quantile%20normalization#quantile-normalization
+
+Below is a simple example of downloading a Dataset using `experiments`:
+
+.. code-block:: shell
+
+    $ refinebio download-dataset --path "~/path/to/dataset/dir/" --email-address "foo@bar.com" --experiments "GSE74410 GSM604796 GSM604797"
+
+Below is a simple example of downloading a Dataset using `dataset_dict`:
+
+.. code-block:: shell
+
+    $ refinebio download-dataset --path "./path/to/dataset.zip" --email-address "foo@bar.com" --dataset-dict '{"GSE74410": ["ALL"]}'
```

Comment:
Could this example include a second experiment with only specific samples listed?

Reply:
You could potentially break it into two lines like:
```
$ dataset_json='{"GSE74410": ["ALL"], ...}'
$ refinebio download-dataset --path "./path/to/dataset.zip" --email-address "foo@bar.com" --dataset-dict $dataset_json
```
If it's getting too long for one line.